src/App.js

```diff
@@ -227,9 +118,6 @@ const returnLoadedWalletComponent = (Component, props, rest) => {
   // Check version
   if (reduxState.isVersionAllowed === undefined && !isServerScreen) {
     const promise = version.checkApiVersion();
```

Comment:
`version.checkApiVersion();` is actually a method that execute many important steps.

```
  checkApiVersion() {
    const newPromise = new Promise((resolve, reject) => {
      const libPromise = hathorLib.version.checkApiVersion();
      libPromise.then((data) => {
        // Update version allowed in redux
        store.dispatch(isVersionAllowedUpdate({allowed: hathorLib.helpers.isVersionAllowed(data.version, hathorLib.constants.MIN_API_VERSION)}));
        // Set network in lib to use the correct address byte
        let network;
        if (data.network === 'mainnet') {
          network = 'mainnet';
        } else {
          // Can we assume it will be testnet? I think it's safe
          network = 'testnet';
        }
        helpers.updateNetwork(network);
        resolve({...data, network});
      }, (error) => {
        reject();
      });
    });
    return newPromise;
  },
```

We don't need to wait for the promise to finish anymore to continue the code but we still need the rest of the code to be executed. I will just remove `const promise`, what do you think? Is this bad practices?

Reply:
I guess you have to at least add a handler in case of error.

Reply:
We already handle error cases in general opening the error modal. Should we do something else?